src/marqo/tensor_search/tensor_search.py

```diff
@@ -1312,7 +1312,7 @@ def _lexical_search(
     # Set the _highlights for each doc as [] to follow Marqo-V1's convention
     if highlights:
         for docs in gathered_docs['hits']:
-            docs['_highlights'] = {}
+            docs['_highlights'] = [{}]
```

Comment:
this needs to be an empty list - an empty dict inside an empty list means that it is no longer an empty list

Reply:
Done